python/interpret_community/__init__.py

```diff
@@ -31,3 +31,8 @@ def close_handler():
         handler.close()
         logger.removeHandler(handler)
     atexit.register(close_handler)
+
+_major = '0'
+_minor = '13'
```

Comment:
did we mean to checkin these versions?

Reply:
Yup, this is necessary to fix a merge conflict (and is the correct version on master rn)